### ISSUE TITLE
Leap prompt is positioned after the 0.10 inlay_hints

### ISSUE THREAD

**Issue by LarssonMartin1998** [ggandor/leap.nvim#228]
When using the new inlay hints that dropped with 0.10, the prompt I get which allows me where to leap isn't positioned next to the jump point if the character is followed by inlay hints, then it is positioned after the inlay hints.

**Comment by ggandor** [comment]
This seems non-trivial to handle. Maybe we should just disable/enable inlay hints on `LeapEnter`/`LeapLeave`, as with certain editor options, e.g. `conceallevel`?

**Comment by LarssonMartin1998** [comment]
> This seems non-trivial to handle. Maybe we should just disable/enable inlay hints on `LeapEnter`/`LeapLeave`, as with certain editor options, e.g. `conceallevel`?

I agree, and I actually already did this in my personal config recently, however, I thought I'd still raise it here. Great minds think alike ;)

Here is [my implementation](https://github.com/LarssonMartin1998/.dotfiles/blob/main/home/.config/nvim/lua/plugs/leap.lua) that toggles inlay hints
